scout/server/blueprints/variants/templates/variants/cancer-variants.html

```diff
@@ -39,7 +39,7 @@
       <div class="card-header">
         <strong><a data-toggle="collapse" data-parent="#accordion" href="#collapseFilters">Filters</a></strong>
       </div>
-      <div class="card-body panel-collapse collapse" id="collapseFilters">
+      <div class="card-body panel-collapse collapse {{ "show" if request.args.get("expand_search") }}" id="collapseFilters" >
```

Comment:
If there is an error in the filter search, redirect and expand the filter search directly because it's likely that the user wants to continue using the filter form. It's annoying having to expand the form once again. This should be the default behavior in all types of variants pages I think

Reply:
:+1: Good idea! We should change that soon so that the pages behave similarly. Open an issue perhaps. 
It is not entirely unlikely that the most common use case for the variants page would be to filter, so perhaps filters should also be open on first visit. I remember HT was a little frustrated over the filter folding away too much when he just started.

Reply:
Issue added